src/index.ts

```diff
@@ -1,61 +1,37 @@
 import path from 'path';
-import diffDirs, { DirsType, DiffResult, ComparisonOptionsType } from './compare';
-import generateReport from './report';
+import generateReportTemplate from './report';
 import fs from 'fs-extra';
 import mkdirp from 'mkdirp';
+import execa from 'execa';
 
-const showResults = ({
-    failed,
-    passed,
-    diffTime,
-}: {
-    failed: number;
-    passed: number;
-    diffTime: number;
-}) => {
-    const results = `
-    ==================
-    Tests failed: ${failed}
-    Tests passed: ${passed}
-    Diff time: ${diffTime} s.
-    `;
-
-    return results;
-};
-
-type onVrtCompleteType = (result: DiffResult, cmpTime: number) => void;
-
-const onVrtCompleteDefaultAction: onVrtCompleteType = (result, cmpTime) => {
-    const info = showResults({
-        failed: result.failed.length + result.missing.length,
-        passed: result.passed.length + result.new.length,
-        diffTime: cmpTime / 1000,
-    });
-
-    console.info(info);
-};
+// Special log messages for TeamCity service. Using by other platforms
+function teamcityLogger(message: string) {
+    console.info(`##teamcity[${message}]`);
+}
 
 export default async function runVrt({
-    teamcity,
     output,
     cwd,
-    onVrtComplete,
     options,
 }: {
-    teamcity: boolean;
     output: string;
     cwd: string;
-    onVrtComplete?: onVrtCompleteType;
```

Comment:
why did you delete this hook? It is for node.js only, but we need to work with data outside of the script scope, please revert this change.